paddle/fluid/pybind/eager_functions.cc

```diff
@@ -160,6 +187,9 @@ PyMethodDef variable_functions[] = {
      METH_VARARGS | METH_KEYWORDS, NULL},
     {"tensor_copy", (PyCFunction)(void (*)(void))eager_api_tensor_copy,
      METH_VARARGS | METH_KEYWORDS, NULL},
+    {"read_next_eagertensor_list",
```

Comment:
`read_next_eagertensor_list` to `read_next_eager_tensor_list`

Reply:
done, thx!